installing/installing_ibm_power/installing-ibm-power.adoc

```diff
@@ -65,6 +65,8 @@ to use an ISO image or network PXE booting to create the machines.
 
 include::modules/installation-user-infra-machines-iso.adoc[leveloffset=+2]
 
+include::modules/installation-user-infra-machines-static-network.adoc[leveloffset=+3]
```

Comment:
As I noted in the comments of this PR, I think this should be added to this assembly as well.